src/main.ts

```diff
@@ -116,6 +120,20 @@ export function main(command?: string) {
         ready: () => AutumnAton.available(),
         sobriety: "either",
       },
+      {
+        name: "June Cleaver",
+        completed: () => !!get("_juneCleaverFightsLeft"),
+        do: $location`Noob Cave`,
+        choices: Object.fromEntries(
```

Comment:
I don't think this will work as expected. this entry is filled in when entering the main() function so it will only run once and set all the entries statically to whatever the initial state KoLmafia is in.

Reply:
@MrFizzyBubbs or @Kasekopf can double check me on this, but I assumed they were allowed to be functions for a reason